### PR TITLE
bug: revert plausible analytics change

### PR DESCRIPTION
**Pull request by w3cj**
* TLDR: we lost about 2 days worth of analytics because some users of the site are hitting a cached back-end function that doesn't have the updated proxy code

* [This error](https://syntax-fm.sentry.io/issues/5146658009/?query=is%3Aunresolved&referrer=issue-stream&statsPeriod=24h&stream_index=3) showed up in sentry
* The error `Could not parse content as FormData.` happens when the [hooks.server.ts](https://github.com/syntaxfm/website/blob/main/src/hooks.server.ts#L110) does not have the updated safe_form_data handle to skip parsing the body as form-data
  * I'm baffled that a back-end function can be cached like this but I guess that's something that can happen on vercel?
 
* I'm reverting my proxy changes to go back to the `vercel.json` file for now

### CODE REVIEW

**Comment by w3cj** [comment]
Confirmed working on the vercel preview deploy.

Don't want to bother anyone on a Sunday, going to make the executive decision and merge. We can chat about this on Monday.
